fluentd/run.sh

```diff
@@ -14,29 +14,74 @@ docker_uses_journal() {
     grep -q "^OPTIONS='[^']*--log-driver=journald" /etc/sysconfig/docker
 }
 
-if [ "${USE_JOURNAL:-}" != false ] ; then
-    if [ -z "${JOURNAL_SOURCE:-}" ] ; then
-        if [ -d /var/log/journal ] ; then
-            export JOURNAL_SOURCE=/var/log/journal
-        else
-            export JOURNAL_SOURCE=/run/log/journal
+if [ "${MUX_ALLOW_EXTERNAL:-}" = "true" ] ; then
+    # mux service implies mux
+    export USE_MUX=true
+fi
+
+if [ "${USE_MUX:-}" != "true" ] ; then
+    if [ "${USE_JOURNAL:-}" != false ] ; then
+        if [ -z "${JOURNAL_SOURCE:-}" ] ; then
+            if [ -d /var/log/journal ] ; then
```

Comment:
are you sure that existence of the directory guarantees that logs go to that dir? why not just listen to both always?